src/redux/actions.ts

```diff
@@ -941,6 +941,29 @@ export function storeNetworkAreaDiagramNodeMovement(
     };
 }
 
+export const STORE_NETWORK_AREA_DIAGRAM_TEXT_NODE_MOVEMENT = 'STORE_NETWORK_AREA_DIAGRAM_TEXT_NODE_MOVEMENT';
+export type StoreNetworkAreaDiagramTextNodeMovementAction = Readonly<
```

Comment:
Maybe reuse the same type `StoreNetworkAreaDiagramNodeMovementAction` defined line 921 instead of duplicate it.
Rename it `NetworkAreaDiagramElementPosition` or something like this